dm/tests/checker/run.md

```diff
@@ -0,0 +1,20 @@
+# manual test
```

Comment:
i think this manual test scriptis need not be included in our repo ?

maybe just prepra a doc and link the doc to PR description ?

Reply:
It is ok for me.